pkg/controller/installation/marketplace/manifestHelper.go

```diff
@@ -0,0 +1,143 @@
+package marketplace
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	manifestEnvVarKey  = "MANIFEST_DIR"
+	defaultManifestDir = "manifests/"
+)
+
+// Generates ConfigMap equivalent from a manifest package
+func GenerateRegistryConfigMapFromManifest(manifestPackageName string) (map[string]string, error) {
+	manifestDir := GetManifestDirEnvVar() + manifestPackageName
+
+	configMapData := make(map[string]string)
+
+	csvStringList, err := GetFilesFromManifestAsStringList(manifestDir, "^*.clusterserviceversion.yaml$", "")
+	if err != nil {
+		logrus.Fatalf("Error proccessing csv packages from %s with error: %s", manifestPackageName, err)
```

Comment:
Can we change this log to state that it couldn't process cluster service versions instead?